src/main/resources/documentation/service-description.html

```diff
@@ -0,0 +1,105 @@
+<div>
+  <h2 id="overview">Overview</h2>
+  <p>A RESTful API service for accessing NOMIS data sets.</p>
+  <p>
+    All times sent to the API should be sent in local time without the timezone e.g. YYYY-MM-DDTHH:MM:SS.
+    All times returned in responses will be in Europe / London local time unless otherwise stated.
+  </p>
+  <p>
+    Some endpoints are described as using the Replica database, a read-only copy of the live database which at
+    time of writing lags by < 1 second up to approximately 2 seconds. These endpoints are not suitable for use
+    by services reacting to events or refreshing web pages where a change has just been made.
+  </p>
+  <p>
+    This service does provide searching capabilities, but can sometimes be slow to return results.  A faster search
+    is available in the
+    <a href="https://prisoner-offender-search-dev.prison.service.justice.gov.uk/swagger-ui/index.html">HMPPS Prisoner Search</a>
+    service.  This also augments the results from the incentives and restricted patients service.  That service should
+    be used in preference, with this service only then being used if the data is not available there.  Furthermore, the
+    Prisoner Search service also provides key prisoner view that could be suitable.
+  </p>
+  <h2 id="role-restrictions">Role restrictions</h2>
+  The following roles are currently in use in this service:
+  <ul>
+    <li>
+      APPROVE_CATEGORISATION
+    </li>
```

Comment:
This seems like a bit of a maintenance headache and will probably drift? Maybe we could create a Gradle task to check that all roles in use are included in this doc, or even add them to this list. Maybe.